README.md

```diff
@@ -12,27 +12,27 @@
 
 ### Redundancy
 
-Akubra is a simple solution to keep an independent S3 storages in sync - almost
+Akubra is a simple solution to keep independent S3 storages in sync - almost
 realtime, eventually consistent.
 
-Keeping synchronized storage clusters, which handles great volume of new objects
-(about 300k obj/h), is the most efficient by feeding them with all incoming data
-at once. That's what Akubra does, with a minimum memory and cpu footprint.
+Keeping synchronized storage clusters, which handle great volume of new objects
+(about 300k obj/h), is most efficient by feeding them with all incoming data
+at once. That's what Akubra does, with a minimal memory and cpu footprint.
 
-Synchronizing S3 storages offline is almost impossible with a high volume traffic.
+Synchronizing S3 storages offline is almost impossible with a high volume of traffic.
 It would require keeping track of new objects (or periodical bucket listing),
-downloading and uploading them to other storage. It's slow, expensive and hard
+downloading and uploading them to the other storage. It's slow, expensive and hard
 to implement robustly.
 
 Akubra way is to put files in all storages at once by copying requests to multiple
-backends. I case one of clusters rejects request it logs that event, and syncronizes
+backends. I case one if clusters rejects request it logs that event, and synchronizes
```

Comment:
If one of the clusters rejects...

Reply:
Good catch, sad that it was after the merge :)